themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -33,6 +33,8 @@ realm-detail.protocol-endpoints.oidc=OpenID Endpoint Configuration
 realm-detail.protocol-endpoints.saml=SAML 2.0 Identity Provider Metadata
 realm-detail.userManagedAccess.tooltip=If enabled, users are allowed to manage their resources and permissions using the Account Management Console.
 userManagedAccess=User-Managed Access
+realm-detail.userDeleteOwnAccountAllowed.tooltip=If enabled, users are allowed to delete their own account from the Account Management Console.
```

Comment:
`realm-detail.userDeleteOwnAccountAllowed.tooltip` can removed actually since we are not using the realm option anymore